front-end/.gitignore

```diff
@@ -2,6 +2,7 @@
 
 # dependencies
 /node_modules
+/uploads
```

Comment:
Confused by this, does electron build directly into the front-end dir with the backend? Doesn't really make sense to gitignore backend things in this dir

Reply:
yes sorry, this was a leftover from testing

Reply:
removed